static/search.js

```diff
@@ -514,16 +514,15 @@ var Search = Control.extend({
 					pathPrefix: (self.options.pathPrefix === '.') ? '' : '/' + self.options.pathPrefix + '/'
 				},{
 					docUrl: function(){
-						if(!self.options.pathPrefix){
-							return this.url;
+						if(self.options.pathPrefix){
+							return self.options.pathPrefix + "/" + this.url;
 						}
-
-						var root = joinURIs(window.location.href, self.options.pathPrefix);
-						if(root.substr(-1) === "/"){
-							root = root.substr(0, root.length-1);
```

Comment:
As far as I can tell it was part of the problem and the new setup forgoes the need to have it at all.